plugin.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 # name: discourse-multilingual
 # about: Features to support multilingual forums
-# version: 0.1.1
+# version: 1.0
```

Comment:
@merefield Why are we jumping straight to 1.0?

Reply:
From https://semver.org:

```
Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
```

So, on the basis we are removing "experimental" I felt we should be leaving 0.x and presenting this as a Production instance.

However, if you prefer we can mark is `0.2.0`